setup.py

```diff
@@ -36,11 +36,15 @@ def read_requirements(filename):
         return [line for line in lineiter if line and not line.startswith("#")]
 
 
+_dask = read_requirements("requirements/dask.txt")
+_dask_cuda12x = read_requirements("requirements/dask_cuda12x.txt")
 _dev = read_requirements("requirements/dev.txt")
 
 requirements = {
-    "base": read_requirements("requirements/base.txt"),
-    "cuda12x": read_requirements("requirements/cuda12x.txt"),
+    "base": read_requirements("requirements/base.txt") + _dask,
+    "base_no_dask": read_requirements("requirements/base.txt"),
+    "cuda12x": read_requirements("requirements/cuda12x.txt") + _dask_cuda12x,
+    "cuda12x_no_dask": read_requirements("requirements/cuda12x.txt"),
```

Comment:
For now, we keep the default installation paths to include Dask dependencies.

I added `base_no_dask` and `cuda12x_no_dask` so that I could test things without Dask being installed.